### PR TITLE
Fix bug 1231732 - Cropped button on Firefox ios page

### PR DESCRIPTION
**Pull request by dhanushuUzumaki**
Fixed cropped button on Firefox ios page

### CODE REVIEW

File: media/css/firefox/ios.less [hunk 1]
```diff
@@ -739,7 +739,7 @@ a.go {
     .font-size(@smallFontSize);
     color: #fff;
     text-align: center;
-    margin-top: 10px;
+    //margin-top: 10px;
```
Comment (alexgibson) [inline media/css/firefox/ios.less]: Please remove what's not needed, as opposed to commenting out individual lines.

**Comment by alexgibson** [comment]
Fix looks good for non-English locales. I think we can keep the default margin for pages in English still however. Adding a selector like this should do it:

    html[lang|="en"] {
        .fxfamilynav-cta-wrapper .send-to {
            margin-top: 10px;
        }
    }

**Comment by dhanushuUzumaki** [comment]
Thanks for the help with lang selector

**Comment by alexgibson** [comment]
I squashed your commits, renamed the commit message and merged in 6c5124e6e8ca72465614cd7b848a589983ce234a

Thanks once again, @dhanushuUzumaki

**Pull request closed without merging**